### DISCUSSION TITLE
cant get access to github education says "You appear not to be near any campus location for the school you have selected." for IGNOU which provides online course.

### DISCUSSION THREAD

**Discussion by CyferBoy** [community/community#102735] [GitHub Education]
### Hi everyone,

I am currently enrolled in IGNOU's online program of post-graduation. and trying to get github student access but it says "You appear not to be near any campus location for the school you have selected." but in an online mode that is not necessory please if anyone have any idea please let me know.
Thanks

**Comment by CyferBoy** [comment]
![image](https://github.com/community/community/assets/70091085/7ab46ab9-79e0-4ec2-9d0f-0bcd5bee6b0a)

**Reply by mayankdedhia** [reply]
@CyferBoy were you able to get this resolved?

**Reply by MadhavKwatra** [reply]
@CyferBoy Any updates here?
